site/src/components/persona.md

```diff
@@ -0,0 +1,156 @@
+| `.persona.persona-<size>` | `xs` `sm` `md (default)` `lg` `xl` | `tablet`, `desktop` |
+
+## Usage
+
+Below are examples of the most common persona component use-cases - one with an image avatar and another with user's initials.
+
+```html
+<div class="persona">
+	<figure class="persona-avatar">
+		<img src="~/src/scaffold/media/avatar.png" alt="username" />
+	</figure>
+	<div class="persona-details">
+		<p class="persona-name">Jane Doe</p>
+		<p>Software Engineer</p>
+	</div>
+</div>
+
+<div class="persona margin-top-md">
+	<div class="persona-avatar">
+		<span>JD</span>
+	</div>
+	<div class="persona-details">
+		<p class="persona-name">John Doe</p>
+		<p>Software Engineer</p>
+	</div>
+</div>
+```
+
+Persona's details may contain a link to user's profile, and with help of the [`.stretched-link` component](../components/stretched-link.md) the whole area persona takes becomes clickable:
+
+```html
+<div class="persona position-relative">
+	<div class="persona-avatar">
+		<span>JD</span>
+	</div>
+	<div class="persona-details">
+		<a class="persona-name justify-self-stretch stretched-link" href="#">John Doe</a>
+		<p>Software Engineer</p>
+	</div>
+</div>
+```
+
+Persona might have both avatar and details parts presented at the same time or just one of them.
+
+```html
+<div class="persona">
+	<figure class="persona-avatar">
+		<img src="~/src/scaffold/media/avatar.png" alt="username" />
+	</figure>
+</div>
+
+<div class="persona margin-top-md">
+	<div class="persona-details">
+		<p class="persona-name">John Doe</p>
+		<p>Software Engineer</p>
+	</div>
+</div>
+```
+
+### Sizes
+
+The following classes are available for resizing avatar:
+
+| Class        | Avatar's size in px |
+| ------------ | ------------------- |
+| `persona-xs` | 28px                |
+| `persona-sm` | 32px                |
+| `persona-md` | 36px                |
+| `persona-lg` | 42px                |
+| `persona-xl` | 48px                |
+
+```html
+<div class="display-flex flex-wrap-wrap gap-xxs">
+	<div class="persona persona-xs">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="username" />
+		</figure>
+	</div>
+	<div class="persona persona-sm">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="username" />
+		</figure>
+	</div>
+	<div class="persona persona-md">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="username" />
+		</figure>
+	</div>
+	<div class="persona persona-lg">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="username" />
+		</figure>
+	</div>
+	<div class="persona persona-xl">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="username" />
+		</figure>
+	</div>
+</div>
+
+<div class="display-flex flex-wrap-wrap gap-xxs margin-top-md">
+	<div class="persona persona-xs">
+		<div class="persona-avatar">
+			<span>ww</span>
```

Comment:
If we're going to support styled-text avatars, we should ensure we demo accessible markup for these:

Could you please add `role=img` and an `aria-label`?

Reply:
Updated